test/data/test_CanMessage.asc

```diff
@@ -2,7 +2,7 @@ date Sam Sep 30 15:06:13.191 2017
 base hex  timestamps absolute
 internal events logged
 // version 9.0.0
-Begin Triggerblock Sam Sep 30 15:06:13.191 2017
+Begin Triggerblock Sat Sep 30 10:06:13.191 PM 2017
```

Comment:
looks like the tests files weren't made with the `ASCWriter` so I updated to make it right. Doesn't appear to mess with any tests.

Reply:
Looks good. Looks like it was generated with a German locale.